Library/ViewModels/LiveStreamCountdownViewModel.swift

```diff
@@ -88,6 +91,10 @@ LiveStreamCountdownViewModelInputs, LiveStreamCountdownViewModelOutputs {
       }
       .map { (day: $0.day ?? 0, hour: $0.hour ?? 0, minute: $0.minute ?? 0, second: $0.second ?? 0) }
 
+    self.countdownDateLabelText = liveStream.map {
+      AppEnvironment.current.dateType.init(timeIntervalSince1970: $0.startDate).date }
+      .map(formattedDateString)
```

Comment:
no need to appeal to the app environment date if you already have a date, can just do `formattedDateString(date: $0.startDate)`.

the app environment date is only needed when you need to construct a new date from scratch.